tools/c7n_policystream/policystream.py

```diff
@@ -862,12 +862,12 @@ def diff(repo_uri, source, target, output, verbose):
         repo_uri = pygit2.discover_repository(os.getcwd())
 
     repo = pygit2.Repository(repo_uri)
-    load_resources()
+    load_available()
 
     # If on master show diff between last commit to current head
     if repo.head.shorthand == 'master':
         if source is None:
-            source = 'master@{1}'
+            source = 'HEAD^1'
```

Comment:
I was getting a key error on master@{1} for the test repo, HEAD^1 seems to be more universal.